dashboard/src/components/AppNew/AppNew.tsx

```diff
@@ -120,6 +149,20 @@ class AppNew extends React.Component<IAppNewProps, IAppNewState> {
                   required={true}
                 />
               </div>
+              <div>
+                <label htmlFor="chartVersion">Version</label>
+                <select id="chartVersion" onChange={this.handleChartVersionChange} required={true}>
+                  {versions.map(v => (
+                    <option
+                      key={v.id}
+                      value={v.attributes.version}
+                      selected={v.attributes.version === this.props.chartVersion}
```

Comment:
warning from js console: ```Warning: Use the `defaultValue` or `value` props on <select> instead of setting `selected` on <option>.```

So we need to change line 154 to add `<select value={this.props.chartVersion}>`.